aiu_fms_testing_utils/utils/quantization_setup.py

```diff
@@ -0,0 +1,147 @@
+# Standard
+from functools import partial
+from typing import Any
+import argparse
+import json
+import os
+
+# Third Party
+from transformers import PreTrainedModel
```

Comment:
make this import gated and fail only if `args.is_quantized` is True in inference.py